test/tsickle_test.ts

```diff
@@ -62,6 +62,9 @@ function compareAgainstGolden(output: string|null, path: string) {
       }
     }
   } else {
+    // Make sure we have proper line endings when testing on Windows.
+    if (golden != null) golden = golden.replace(/\r\n/g, '\n');
+    if (output != null) output = output.replace(/\r\n/g, '\n');
```

Comment:
Line 54 writes the file on disk. On Windows this would mean `\n` in written files ... but now that I think about it, `output !== golden` in line 51 is always false on Windows. I'll change it.